tools/apkanalyser/result/apk-analyser-result.xsl

```diff
@@ -0,0 +1,214 @@
+      footer  { border: 0px solid #eee; padding: 10px 0px; margin-botton: 10px; text-align: right; }
+      table { width: 100%; text-align: center; border-collapse:collapse; font-size: 12px; }
+      th, td { border-bottom: 1px solid #eee; padding: 3px 0px; overflow: hidden;}
+      tr:hover { background-color: #fafafa; cursor: pointer;} 
+      td.details { background-color: rgba(32, 50, 68,1.0); color: rgba(255, 255, 255, 0.6);}
+      td.details:hover { background-color: rgba(20, 38, 56,1.0);  }
+      td.details, .smali, .asset { text-align: left; margin: 20px 0px ; }
+      span.cap1, .smali span { display:inline-block; margin: 2px 4px; border: 1px dotted rgba(255, 255, 255, 0.6); padding: 1px 6px; border-radius: 0px; rgba(255, 255, 255, 0.9); }
+      span.cap1, .smali span:hover { border: 1px solid rgba(255, 255, 255, 1.0); background-color: rgba(142, 68, 173,1.0); color: rgba(255, 255, 255, 1.0);  }
+      span.cap2, .asset span { display:inline-block; margin: 2px 4px; border: 1px dotted rgba(255, 255, 255, 0.6); padding: 1px 6px; border-radius: 10px; rgba(255, 255, 255, 0.9); }
+      span.cap2, .asset span:hover { border: 1px solid rgba(255, 255, 255, 1.0); background-color: rgba(211, 84, 0,1.0); color: rgba(255, 255, 255, 1.0); }
+      .det { margin: 0px 10px 0px 0px; font-weight: bold; text-shadow: 0px 1px 1px rgba(0,0,0,0.1); font-size: 13px; color: rgba(255, 255, 255, 1.0); }
+      .apd { text-align: center;  margin-top: 10px; }
+      .details { padding: 10px; font-size:11px; }
+      .fill {background-color: #46C8C8; color: rgba(255, 255, 255, 1.0); }
+      .lim { min-width: 100px; max-width: 200px; overflow: hidden; }
+      .limname { max-width: 120px; overflow: hidden;  }
+      .limsize { min-width: 62px; max-width: 80px; overflow: hidden;  }
+      .limw { min-width: 60px; max-width: 106px; overflow: hidden; width: 106px;  }
+      .emb { background-color:#00B1E1; color: rgba(255, 255, 255, 1.0); text-shadow: 0px 1px 0px rgba(0,0,0,0.2);}
+      .sha { background-color:#EB7E7F; color: rgba(255, 255, 255, 1.0); }
+      .arm { background-color:#EC795E; color: rgba(255, 255, 255, 1.0);}
+      .armx86 { background-color:#E082A5; color: rgba(255, 255, 255, 1.0);}
+      .x86 { background-color:#FFBE65; color: rgba(0, 0, 0, 0.6); }
+      .rt { background-color:#0ACDC7; color: rgba(255, 255, 255, 1.0); }
+      .cdo { background-color:#A7D773; color: rgba(255, 255, 255, 1.0); }
+      .ci { background-color:#FF885E; color: rgba(255, 255, 255, 1.0); }
+      .xdk { background-color:#D18EE2; color: rgba(255, 255, 255, 1.0); }
+      #datetime { font-size: 10px; margin-top: 10px; }
+      #toggle { text-align: right; color: rgba(22, 160, 133,1.0); cursor: hand; margin: -6px 0px 6px 0px;  }
+      #toggle:hover { color: rgba(16, 154, 127,1.0); }
+    </style>
+    <script type='text/javascript' src='https://code.jquery.com/jquery-2.1.3.min.js'></script>
```

Comment:
Is ` type='text/javascript'` a must?

Please don't involve a 3rd-party site. How about `tools/resources/webrunner/jquery-1.10.2.min.js`?

Hmm, I have been considering to unify the jquery.js at https://github.com/crosswalk-project/crosswalk-test-suite/issues/875#issuecomment-68036384